shared/actions/chat2/index.js

```diff
@@ -1289,6 +1289,8 @@ const changeSelectedConversation = (
           ),
           ...(isMobile ? [Saga.put(Chat2Gen.createNavigateToThread())] : []),
         ])
+      } else if (isMobile) {
```

Comment:
related to stuff @akalin is doing. if we set pendingMode === none we want to go back to the inbox and not leave ourselves in the thread